parsers/verilog.c

```diff
@@ -1355,13 +1355,16 @@ static int processParameterList (tokenInfo *token, int c)
 	return c;
 }
 
-// [ virtual ] class [ static | automatic ] class_identifier [ parameter_port_list ]
+// [ virtual ] class [ : final ] class_identifier [ parameter_port_list ]
```

Comment:
Generally, ctags should accept older syntax, too.
(Your change doesn't reject the older syntax. This is good.)
Keeping the comment explaining older syntax is better.
Just adding the comment explaining the new syntax is enough:
```
// LRM 19??
//  [ virtual ] class [ static | automatic ] class_identifier [ parameter_port_list ]
// LRM 20??
//  [ virtual ] class [ : final ] class_identifier [ parameter_port_list ]
```

Reply:
Thanks for your suggestion, I will fix it.